pages/mentored_mode_vs_independent_mode.md

```diff
@@ -10,10 +10,14 @@ Every time you complete a solution, you'll also unlock extra bonus exercises. Th
 
 We try to respond to all core submissions within 24hrs, but as our mentors are volunteers helping in their evenings and weekends, we can't guarantee that we will hit that goal. Our bonus exercises hopefully mean you won't get frustrated waiting for feedback :)
 
-### Independent Mode
+### Practice Mode
 
-We designed Independent Mode for people who want to take fast, deep-dives into a language without waiting for mentor feedback. In Independent Mode you can take the exercises in whatever order you want and there is no progression or unlocking. You may request mentoring on one exercise at a time, but your solutions will have a lower priority than people in Mentored Mode.
+We designed Practice Mode for people who want to take fast, deep-dives into a language without waiting for mentor feedback. In Practice Mode you can take the exercises in whatever order you want and there is no progression or unlocking. You may request mentoring on one exercise at a time, but your solutions will have a lower priority than people in Mentored Mode.
 
 ### Changing mode on a track
 
-While we recommend sticking to one mode per track, it is possible to change if you decide you've made the wrong decision. You will find the button in the "Danger Zone" section of a track page, which is on the right-hand-side in Mentored Mode and the bottom of the page in Independent Mode.
+While we recommend sticking to one mode per track, it is possible to change if you decide you've made the wrong decision. You will find the button in the "Danger Zone" section of a track page, which is on the right-hand-side in Mentored Mode and the bottom of the page in Practice Mode.
```

Comment:
As long as the "Danger Zone" remains well defined, should we state where on the page it is?  If the style changes, or even if the layout changes, we would have to remember to change this location.  And since some folks have been doing custom styles (stylish, anyone?) I don't think we can guarantee this is accurate.

Reply:
This PR doesn't introduce or change the content of the Danger Mode bit so I'd rather open a separate discussion for that, rather than it blocking this one.

Reply:
I guess that is the problem with not having "one sentence per line" commits.  I saw green and didn't look much closer.